data/content/about.md

```diff
@@ -1,3 +1,51 @@
-Free and guided workshops in Rust – a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+RustBridge is a workshop focused on getting underrepresented people with
+a background in another programming language to learn Rust and join the
+community.
 
-Our events are aimed at people who are underrepresented in technology and offer the opportunity to learn within a friendly, safe and sometimes fun environment. We do this by providing rewarding learning experiences, building helpful tools as well as materials and gathering a group of awesome humans. RustBridge is a project by the Rust Language Community Team.
+### Why RustBridge?
+
+RustBridge is an excellent way to welcome and attract persons from
+underrepresented backgrounds to your Rust conference or event. By giving
+students new to Rust an introduction to the language and other fellow Rust
+learners before the conference, RustBridge increases the relevance and
+value of Rust event content for students, as well as offering students a
+sense of belonging and camaraderie, both of which are things new members
+to a community often lack. Students of previously run Bridge programs often
+cite the Bridge Program as the reason they felt comfortable attending the
+event, and that the program greatly increased their enjoyment of the event.
+
+### Why Rust?
+
+[Rust is a programming language] designed to improve the experience of systems
+programming. Systems programming is notoriously difficult and learners are
```

Comment:
huh yeah that's interesting. i def struggled with this sentence. i am totally open to suggestions, but may land this for the time being in the interest of having something, if we don't know how to immediately update it. always open to changes and refinement!